spec/spec_helper.rb

```diff
@@ -0,0 +1,96 @@
+# This file was generated by the `rails generate rspec:install` command. Conventionally, all
```

Comment:
Can we get rid of the comments in this file?

They clutter the place up.

(ditto throughout)

Reply:
Sure